euth/actions/emails.py

```diff
@@ -19,8 +21,10 @@ def notify_creator_on_create_action(action):
         'url': url,
     }
 
+    jsonDec = json.decoder.JSONDecoder()
+
     emails.send_email_with_template(
-        [action.target.creator.email], 'notify_creator', context)
+        jsonDec.decode(action.recipients), 'notify_creator', context)
```

Comment:
This will sent one email with all recipients in the `To:`-header, I think we should not do this.